test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextTests.cs

```diff
@@ -11,9 +12,9 @@ namespace Microsoft.Extensions.DependencyModel.Tests
     public class DependencyContextTests
     {
         [Theory]
-        [InlineData("System.Banana.dll", "System.Banana")]
-        [InlineData("System.Banana.ni.dll", "System.Banana")]
-        [InlineData("FlibbidyFlob", "FlibbidyFlob")]
+        [InlineData("System.Collections.dll", "System.Collections")]
+        [InlineData("System.Collections.ni.dll", "System.Collections")]
+        [InlineData("mscorelib", "mscorelib")]
```

Comment:
Looks like the names don't actually matter for this test, but yeah, it would be best to be consistent and use `mscorlib`